.github/workflows/release.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         uses: gmiam/rust-musl-action@9e6a37bf27ecfffb6b92240ea276bea5487fa15d
         continue-on-error: false
         with:
-          args: cargo-deb && rustup target add $BUILD_TARGET
+          args: cargo install cargo-deb && cargo deb && rustup target add $BUILD_TARGET
```

Comment:
Why is the `&& dustup target add ..` needed?

Reply:
since we want to incorporate it into the build target, but it wasn't working with simple --target $BUILD_TARGET, when I ran the workflow. it suggested me to use rustup add target, and when I used it, it started working

Reply:
I used "cargo deb --target=$BUILD_TARGET", which is supported by cargo deb, but it didn't work, and then I used the "rustup target add $BUILD_TARGET" ,then it worked.

Reply:
`cargo deb` is the second step in your chain. It creates the deb file. If you have already created the deb file why do you need to install anything. Do you usually eat food and then cook it, or cook it before eating?